crates/viewer/re_viewer/src/ui/welcome_screen/welcome_section.rs

```diff
@@ -10,6 +10,19 @@ pub(super) const WELCOME_SCREEN_BULLET_TEXT: &[&str] = &[
     "Configure the viewer interactively or through code",
 ];
 
+// XXX TODO MOVE MACRO TO UTILITY MODULE OR UTILITY CRATE
+macro_rules! color_from_rgb_hex {
+    ($hex:literal) => {
+        match color_hex::color_from_hex!($hex).as_slice() {
+            [a, b, c] => Color32::from_rgb(*a, *b, *c),
+            _ => panic!("XXX"),
+        }
+    };
+}
```

Comment:
I think this should be defined in `ecolor`, I should be able to propose this over the weekend.

Reply:
There already is an `ecolor::hex_color!` macro: https://docs.rs/ecolor/latest/ecolor/macro.hex_color.html